mmf/modules/metrics.py

```diff
@@ -459,6 +459,22 @@ def get_gt_ranks(self, ranks, ans_ind):
             gt_ranks[i] = int(ranks[i, ans_ind[i].long()])
         return gt_ranks
 
+    def process_ranks(self, ranks):
+        num_opts = 100
+
+        # none of the values should be 0, there is gt in options
+        if torch.sum(ranks.le(0)) > 0:
+            num_zero = torch.sum(ranks.le(0))
+            print(f"Warning: some of ranks are zero: {num_zero}")
```

Comment:
let's try to use the warning module, and not have print statements.

Reply:
That's a great idea, just changed it :) thanks!